### PR TITLE
Add missing `utf8` option for ftp adapter

### PR DESCRIPTION
**Pull request by mynameisbogdan**
Adding the missing `utf8` option for the ftp adapter.

### CODE REVIEW

File: src/Adapter/Builder/FtpAdapterDefinitionBuilder.php [hunk 1]
```diff
@@ -60,6 +60,9 @@ protected function configureOptions(OptionsResolver $resolver)
 
         $resolver->setDefault('ignore_passive_address', null);
         $resolver->setAllowedTypes('ignore_passive_address', ['null', 'bool']);
+
+        $resolver->setDefault('utf8', false);
+        $resolver->setAllowedTypes('utf8', 'scalar');
```
Comment (tgalopin) [inline src/Adapter/Builder/FtpAdapterDefinitionBuilder.php]: Could you use `bool`?

File: src/Adapter/Builder/FtpAdapterDefinitionBuilder.php [hunk 2]
```diff
@@ -60,6 +60,9 @@ protected function configureOptions(OptionsResolver $resolver)
 
         $resolver->setDefault('ignore_passive_address', null);
         $resolver->setAllowedTypes('ignore_passive_address', ['null', 'bool']);
+
+        $resolver->setDefault('utf8', false);
```
Comment (tgalopin) [inline src/Adapter/Builder/FtpAdapterDefinitionBuilder.php]: I didn't have a deep look but I wasn't able to find the default value in Flysystem for this option. Are you use it's false? If it's not, this is a BC break.

File: src/Adapter/Builder/FtpAdapterDefinitionBuilder.php (hunk 1 above)
Comment (mynameisbogdan) [reply src/Adapter/Builder/FtpAdapterDefinitionBuilder.php]: Ok, I updated it with type `bool`.

File: src/Adapter/Builder/FtpAdapterDefinitionBuilder.php (hunk 2 above)
Comment (mynameisbogdan) [reply src/Adapter/Builder/FtpAdapterDefinitionBuilder.php]: Well, in `League\Flysystem\Adapter\Ftp` the default value is set with `protected $utf8 = false;`.

**Comment by tgalopin** [comment]
Thanks you @mynameisbogdan !

**Pull request merged**